code/modules/mob/living/carbon/human/species.dm

```diff
@@ -55,6 +55,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/allow_numbers_in_name // Can this species use numbers in its name?
 	var/datum/outfit/outfit_important_for_life /// A path to an outfit that is important for species life e.g. plasmaman outfit
 	var/datum/action/innate/flight/fly //the actual flying ability given to flying species
+	var/stance
```

Comment:
Why is it on human species not grod species?

Reply:
kapu code, moved in.